draft-demarco-oauth-status-assertions.md

```diff
@@ -782,6 +901,13 @@ Status Assertions are based on a privacy-by-design approach, reflecting
 a deliberate effort to balance security and privacy needs in the
 Digital Credential ecosystem.
 
+## Validity Reasons
+
+Status assertions can reveal information about the holder or subject that was not commited to in the original credential issuance.
```

Comment:
```suggestion
Status Assertions may disclose details about the Holder or subject that were not initially committed to during the original Credential issuance. This can potentially expose additional information that was not part of the original credentialing process.
```

ok, we need to align the general requirements mandating not having subject's information within the assertions